prebid-server/developers/add-new-bidder-go.md

```diff
@@ -883,9 +933,9 @@ This chapter will guide you through the creation of automated unit tests to cove
 
 ### Adapter Code Tests
 
-Bid requests and server responses can be quite verbose. To avoid large blobs of text embedded within test code, we've created a framework for bid adapters which use a JSON body and/or a url. If your bidding server uses another payload format, such as XML, you're on your own.
+Bid requests and server responses can be quite verbose. To avoid large blobs of text embedded within test code, we've created a framework for bid adapters which use a JSON body and/or a url. If your bidding server uses another payload format, such as XML, you're on your own. Prebid Server core also makes use of JSON tests to find data race conditions in your adapter code. Hence, they are highly encouraged over coded tests or tests with other payload formats.
```

Comment:
I'd like to avoid the use of the "race" terminology if possible. I also independently wrote an update, what do you think of:

Bid requests and server responses can be quite verbose. To avoid large blobs of text embedded within test code, we've created a framework for bid adapters which use a JSON body and/or a url to send a bid request. We require the use of our test framework as it includes checks to ensure no changes are made to shared memory.

Reply:
Looks fantastic. Modified.